docs/pages/Cardinal/API-Reference/cql.mdx

```diff
@@ -0,0 +1,54 @@
+# Component Query Language
+
+## Language Description:
+
+CQL (Component Query Language) is a simple query language that queries entities based on components contained in the entity.
+
+In the language, you have components, which is simply the names of the specified components.
+
+Examples:
+
+* healthComponent
+* attackComponent
+* armComponent
+* legComponent
+
+You also have two functions which can be called with a variadic amount of components as parameters:
+
+`EXACT(…) and CONTAINS(…)`
```

Comment:
maybe here we can add a header, Functions, and each sub-header can be the function types, contains/exact, followed by description + examples